internal/cloud/cloud_username_manager.go

```diff
@@ -63,6 +126,15 @@ func (c *CloudUsernameManager) OnChange(ctx context.Context, st store.RStore) {
 	currentlyMakingRequest := c.currentlyMakingRequest
 	c.mu.Unlock()
 
+	// if a refresh has been induced, then do a long_get lookup, so that we get the username as soon as
+	// the user has finished the process
```

Comment:
see also: note about naming "refresh", but this comment is confusing. consider like, `state.NeedTiltCloudUsernameAfterRegister` and:
> If the user is currently registering a token, then do a long_get... has finished the registration process